project/Dependencies.scala

```diff
@@ -18,6 +18,8 @@ object Dependencies {
   val mavenApiVersion = "3.5.2"
   val mavenAnnotationsVersion = "3.5"
   val mavenScalaPluginVersion = "3.2.2"
+  val gradleVersion = "4.8.1"
```

Comment:
I took the liberty of updating this to 4.8.1, 3.0.0 is very outdated.

Reply:
I chose 3.0 because that is the oldest version compatible. By compiling it with `3.0` it is compatible with all versions above 3  including the latest `4.8.1` (I tried it). I also asked a friend who works for Gradle about this and he recommended to choose the oldest one we would like to support.

Reply:
Oh interesting! So did your friend recommend you 3.0 for testing the gradle API or for testing the plugin itself? I'm fine leaving this for 3.0, but I'd like to test the plugin with the latest gradle version so that we are closer to the versions that our users will be at.

Reply:
I think the best would be to compile with the oldest version possible to cover as much versions as we can, but always test it with the latest (or with both). 
My fear of people not having the latest Gradle version may be a bit subjective as we at Prezi are still stuck with 2.13 :) But covering more versions without extra effort would no hurt, I think.